docs/source/recipes/recipes_overview.rst

```diff
@@ -0,0 +1,70 @@
+.. _recipes_overview_label:
+
+================
+Recipes Overview
```

Comment:
I personally like having a little context so this can be a kind-of standalone document for a reader. Second opinions? @joecummings @RdoubleA @ebsmothers @pbontrager

Reply:
Personally I like it. Without it there we just jump right into individual recipe pages and it's not really clear what their purpose is; I feel like this page provides useful framing for the entire section.